<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>750</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bgwidget" native="true">
     <property name="minimumSize">
      <size>
       <width>1400</width>
       <height>750</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#bgwidget{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(170, 170, 255, 255), stop:1 rgba(94, 207, 255, 255));
}

</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <item row="24" column="1" colspan="2">
         <widget class="QPushButton" name="signup2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>61</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 174, 174);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
color:rgb(255, 255, 255);
</string>
          </property>
          <property name="text">
           <string>SIGN UP</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="24" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="25" column="1" colspan="2">
         <widget class="QPushButton" name="backtologin">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>61</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(255, 174, 174);
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
color:rgb(255, 255, 255);




</string>
          </property>
          <property name="text">
           <string>Back to login</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="signupuser">
          <property name="minimumSize">
           <size>
            <width>611</width>
            <height>61</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);
margin: 2px;
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 14pt &quot;Calibri&quot; bold;

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="24" column="4">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="25" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="confirmpass">
          <property name="minimumSize">
           <size>
            <width>611</width>
            <height>61</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);
margin: 2px;
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 14pt &quot;Calibri&quot; bold;

</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="signuppass">
          <property name="minimumSize">
           <size>
            <width>611</width>
            <height>61</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);
margin: 2px;
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 14pt &quot;Calibri&quot; bold;

</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="signup1">
          <property name="styleSheet">
           <string notr="true">color:rgb(255, 85, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="25" column="3">
         <layout class="QHBoxLayout" name="horizontalLayout_2"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>81</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(49, 10, 149);
font: 35pt &quot;Courier New&quot;;
</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Sign up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="signupname">
          <property name="minimumSize">
           <size>
            <width>611</width>
            <height>61</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 0, 0, 0);
margin: 2px;
border:1px solid rgb(0, 55, 80);
color: rgb(0, 55, 80);
font: 75 14pt &quot;Calibri&quot; bold;

</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
